### PR TITLE
PUPIL-457 Fix/start lesson label bug

### PR DESCRIPTION
**Pull request by benprotheroe**
## Description

Music year: 1951

overview next section label wasnt being driven by lesson state.lessonStarted

now using lesson sate to dictate label

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2289--oak-web-application.netlify.thenational.academy/pupils/l/shakespearean-comedy-the-tempest-88f0/lessons/context-of-the-tempest-cctp4c
2. Click through lesson
3. You should see once intro is complete the label should say continue lesson untilo the lesson is finished

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -41744,15 +41744,6 @@
         "url": "https://github.com/sponsors/isaacs"
       }
     },
-    "node_modules/npm/node_modules/glob/node_modules/minipass": {
```
[the remark this answers is not in the document]
Comment (kimon-satan) [reply package-lock.json:41747]: might be worth running npm i, seeing if anything changes and committing again if necessary.

**Review by kimon-satan — APPROVED** [review]
nice simple fix. Just check that dodgy package.lock

**Review by rightsaidjames — APPROVED** [review]
From a user perspective I much prefer this logic and I think it's more consistent/easy to understand.

It fixes the original bug and also provides a better experience when taking the lesson in the 'wrong' order.

Screenshots:

Screenshot 2024-02-29 at 16 39 33
Screenshot 2024-02-29 at 16 40 16
Screenshot 2024-02-29 at 16 41 34

**Pull request merged**